logdna/request.go

```diff
@@ -76,64 +67,16 @@ func (c *requestConfig) MakeRequest() ([]byte, error) {
 	req.Header.Set("servicekey", c.serviceKey)
 	res, err := c.httpClient.Do(req)
 	if err != nil {
-		return nil, fmt.Errorf("Error during HTTP request: %s, %+v", err, c)
+		return nil, fmt.Errorf("Error during HTTP request: %s", err)
 	}
 	defer res.Body.Close()
 
 	body, err := c.bodyReader(res.Body)
 	if err != nil {
-		return nil, fmt.Errorf("Error parsing HTTP response: %s, %+v", err, c)
+		return nil, fmt.Errorf("Error parsing HTTP response: %s, %s", err, string(body))
```

Comment:
We don't wanna expose the `servicekey` which is part of `c`.  That shouldn't be needed to report the error.